### DISCUSSION TITLE
Build Application using editor

### DISCUSSION THREAD

**Discussion by JedS** [o3de/o3de#14022] [General]
Hi,
We currently develop a software using OpenScenegraph and Qt.
Ones of the features of our software let the users to build geometries with our CAD editor.
Users can :
o	Adding new shapes (triangle, quadrangle, rectangle, sphere, torus, …)
o	Position the shape with manipulator ( gizmo, translation, rotation, homeotic, …)
o	Make some boolean operation on shapes with the use of cutters
o	Add and define new polygonal meshes over the geometry.

These features are very close to the feature expose by the current O3DE editor

So my question is : Can we embedd part of the O3DE editor in our software ? 

Regards

**Comment by AMZN-daimini** [comment]
If your question relates to licensing, O3DE's licenses are Apache 2.0 or MIT, so as long as your use complies with either you should be good.
If it's technical, I guess it should be possible but it's hard to say without more details :)

**Reply by JedS** [reply]
My question is not related to the license but is technical. Some engines like Unreal or Unigine provide an editor but their API does not allow to embed parts of the editor (asset brower, gizmo manipulator, transformation, ...) in a client application.
I was wondering if this was possible with O3DE?
